Tests failed. Not sure if it's related to these changes or not:
```
======================================================================
FAIL: testMultipleCases (casexml.apps.phone.tests.test_sync_mode.SyncTokenUpdateTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/casexml/apps/phone/tests/restore_test_utils.py", line 45, in inner
    return helper(*args, **kwargs)
  File "corehq/ex-submodules/casexml/apps/phone/tests/restore_test_utils.py", line 32, in __call__
    call_with_settings(fn_with_pre_and_post, run_config.settings, args, kwargs)
  File "/home/travis/build/dimagi/commcare-hq/corehq/apps/tzmigration/test_utils.py", line 28, in call_with_settings
    fn(*args, **kwargs)
  File "corehq/ex-submodules/casexml/apps/phone/tests/restore_test_utils.py", line 28, in fn_with_pre_and_post
    self.fn(*args, **kwargs)
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py", line 204, in testMultipleCases
    self._testUpdate(sync_log.get_id, {"asdf": []})
  File "corehq/ex-submodules/casexml/apps/phone/tests/test_sync_mode.py", line 100, in _testUpdate
    self.assertEqual(set(all_ids), sync_log.case_ids_on_phone)
AssertionError: set(['asdf']) != JsonSet([])
```

--- a/corehq/form_processor/interfaces.py
+++ b/corehq/form_processor/interfaces.py
@@ -166,7 +166,7 @@ class FormProcessorInterface(object):
     def submit_form_locally(instance, domain='test-domain', **kwargs):
         from corehq.apps.receiverwrapper.util import submit_form_locally
         response, xform, cases = submit_form_locally(instance, domain, **kwargs)
-        # response is an iterable to @to_generic doesn't work
+        # response is an iterable so @to_generic doesn't work
         return response, xform.to_generic(), [case.to_generic() for case in cases]
 
     @staticmethod

